horace_core/sqw/page_operations/PageOpBase.m

```diff
@@ -1,9 +1,17 @@
 classdef PageOpBase
-    %PAGEOPBASE class defines generic operation, performed on chunk of pixels
+    % PAGEOPBASE class defines generic operation, performed on chunk of pixels
     % located in memory.
```

Comment:
```suggestion
    % PAGEOPBASE class wraps generic operations to be performed on chunks of pixels
```

Reply:
its not wrapping the operation. It defines it. And interface to perform the operation.